templates/CRM/Contract/Form/PaymentPreview/sepa_mandate.tpl

```diff
@@ -50,7 +50,7 @@
         // Next debit
         const action = CRM.vars["de.systopia.contract"].action;
         const cycleDay = formFields["pa-sepa_mandate-cycle_day"].val();
-        const startDate = formFields["activity_date"].val();
+        const startDate = formFields["start_date"].val();
```

Comment:
You might have already noticed/fixed this locally, but just in case: I'm getting `Uncaught TypeError: Cannot read property 'val' of undefined` on this line when testing the update form for a SEPA or PSP contract. (Randomly noticed this while testing for GP-14437.)

Reply:
Yes, thanks! I have already fixed this locally.